packages/components/select-v2/src/select.vue

```diff
@@ -226,7 +226,7 @@
               <component :is="clearIcon" />
             </el-icon>
             <el-icon
-              v-if="validateState && validateIcon"
+              v-if="validateState && validateIcon && needStatusIcon"
```

Comment:
Perhaps we can consider adjusting the judgment condition here and add a third logical judgment only when `elForm` exists?

Reply:
Now this line does not need to be adjusted, the `statusIcon` can only be configured when used in the Form